src/Uno.UI/UI/Xaml/Controls/Image/Image.Android.cs

```diff
@@ -512,48 +508,22 @@ private async Task SetSourceDrawableAsync(CancellationToken ct, ImageSource newI
 			OnImageOpened(newImageSource);
 		}
 
-		private void SetSourceBitmap(ImageSource newImageSource)
+		private void SetSourceBitmap((ImageSource src, Bitmap data) image)
 		{
 			if (MustDispatchSetSource())
 			{
-				Dispatch(ct => SetSourceBitmapAsync(ct, newImageSource));
+				Dispatch(ct => SetSourceBitmapAsync(ct, image));
 			}
 			else
 			{
-				var unused = SetSourceBitmapAsync(CancellationToken.None, newImageSource);
+				var unused = SetSourceBitmapAsync(CancellationToken.None, image);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'unused'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5633443)